web/vital_records/views/birth.py

```diff
@@ -1,8 +1,12 @@
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import never_cache
+
 from web.vital_records.forms.birth import CountyForm, NameForm
 from web.vital_records.views import common
 from web.vital_records.mixins import Steps
 
 
+@method_decorator(never_cache, name="dispatch")
 class NameView(common.NameView):
```

Comment:
I note this decorator is applied here to the `birth.NameView` _and_ `common.NameView` down below. It is needed on both?